sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/MonotonicallyIncreasingID.scala

```diff
@@ -85,7 +86,8 @@ case class MonotonicallyIncreasingID() extends LeafExpression with Stateful {
       $countTerm++;""", isNull = FalseLiteral)
   }
 
-  override def prettyName: String = "monotonically_increasing_id"
+  override def prettyName: String = getTagValue(
```

Comment:
If we override `nodeName` here, the default implementation of `prettyName` takes care of the alias tag.